src/code/client.ts

```diff
@@ -497,20 +497,37 @@ class CloudFileManagerClient {
     }
   }
 
+  /**
+   * Disassociates the current document from its provider.
+   *
+   * This is important specifically for autosaving providers. When authenticated
+   * state is lost we must not continue to autosave.
+   */
+  disconnectCurrentFile() {
+    console.log('Closing file (rejected reauth)');
```

Comment:
```suggestion
    console.warn('Closing file (rejected reauth)');
```
My convention is to use `console.warn()` for events worth noting in the logs on production and `console.log()` for events used locally when debugging but that should be removed before releasing. This seems like an event worth noting in the production logs.